README.md

```diff
@@ -179,13 +179,18 @@ A glossary entry is structured like this:
 -   **Why use Jekyll for the online version?**
     It is the default for GitHub Pages.
 
-## Collaborators
+## Funding and Collaborators
 
 SADiLaR is one of the collaborators in the finalisation and expansion of the Glosario
 Project to African Languages. SADiLaR is a research infrastructure established by the
 Department of Science and Innovation of the South African government as part of the
 South African Research Infrastructure Roadmap (SARIR).
 
+To advance data science knowledge and accessibility for our diverse community, we developed Glosario. 
```

Comment:
I'm not sure we need this sentence here - it would be more suited to the introduction at the top of the page?